Library/Formula/go.rb

```diff
@@ -18,10 +18,16 @@ class Go < Formula
   option "with-cc-all", "Build with cross-compilers and runtime support for all supported platforms"
   option "with-cc-common", "Build with cross-compilers and runtime support for darwin, linux and windows"
   option "without-cgo", "Build without cgo"
+  option "without-godoc", "godoc will not be installed for you"
 
   deprecated_option "cross-compile-all" => "with-cc-all"
   deprecated_option "cross-compile-common" => "with-cc-common"
 
+  resource "godoc" do
+    url "https://go.googlesource.com/tools.git",
+    :revision => "69db398fe0e69396984e3967724820c1f631e971"
```

Comment:
No tags at all. The best I can do on that is to checkout the ` release-branch.go1.4 ` branch instead of a recent commit reference.